mpf/modes/high_score/code/high_score.py

```diff
@@ -148,7 +148,7 @@ def _run(self) -> Generator[int, None, None]:
                     new_list.append((player, player[category_name]))
 
             # sort if from highest to lowest
-            new_list.sort(key=lambda x: x[1], reverse=True)
+            new_list.sort(key=lambda x: x[1], reverse=(not self.high_score_config['reverse_sort']))
```

Comment:
I guess we could go with ``category_name in self.high_score_config['reverse_sort']`` (or similar) here. That would allow a setting per category. Usually you want the normal score to be sorted normally (highest score first). However, scores such as "time to wizard mode" would be reverse sorted. That is why I would put this per category.

Config would then look like this:

```
high_score:
  _overwrite: True
  categories: !!omap
  - score:
      - GRAND CHAMPION
      - HIGH SCORE 1
      - HIGH SCORE 2
      - HIGH SCORE 3
      - HIGH SCORE 4
  - loops:
      - LOOP CHAMP
  - time_to_wizard:
      - FASTEST WIZARD
      - ALMOST FASTEST WIZARD
  defaults:
    score:
      - BRI: 4242
      - GHK: 2323
      - JK: 1337
      - QC: 42
      - MPF: 23
    loops:
      - JK: 42
    time_to_wizard_mode:
      - JK: 300
      - BM: 350
  reverse_sort:
     - time_to_wizard_mode
```

Makes sense?

Reply:
yes, this cleared it up for me, thank you.